backend/spec/requests/api/v1/favourite_investors_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "swagger_helper"
+
+RSpec.describe "API V1 Favourite Investor", type: :request do
+  path "/api/v1/investors/{investor_id}/favourite_investor" do
+    post "Mark Investor as favourite" do
+      tags "Investors"
+      consumes "application/json"
+      produces "application/json"
+      security [csrf: [], cookie_auth: []]
+      parameter name: :investor_id, in: :path, type: :string
+      parameter name: :empty, in: :body, schema: {type: :object}, required: false
+
+      let(:investor) { create :investor }
+      let(:investor_id) { investor.id }
+      let(:user) { create :user, account: create(:account, :approved) }
+
+      it_behaves_like "with not authorized error", csrf: true
+      it_behaves_like "with forbidden error", csrf: true
```

Comment:
Maybe we need to refactor specs to make those behaviors more clear. Like `it_behaves_like "with forbidden error", user: -> { create(:user) }`. To know for what kind of user we are testing.

Reply:
Like it, will do it :)